packages/calling/calling/src/common/Utils.ts

```diff
@@ -289,6 +289,176 @@ export function emitFinalFailure(emitterCb: LineErrorEmitterCallback, loggerCont
  *
  */
 export async function handleRegistrationErrors(
+  err: WebexRequestPayload,
```

Comment:
dont we have a generic code for each of these errors ?

Reply:
We have a generic code for each type of error we're passing in here. We're handling different error codes in the switch statement of this method.